UWA-1848/Endpoint to block a phone

bmoelans opened the pull request:
https://vikingco.atlassian.net/browse/UWA-1848

bmoelans commented on definitions/telco/block_phone_request.yaml:
```diff
@@ -0,0 +1,18 @@
+type: object
+title: Phone block request
+required:
```
Not sure yet the other attributes are optional, asked care but waiting for info

freeduerinckx commented on paths/telco/block_phone.yaml:
```diff
@@ -0,0 +1,27 @@
+  tags:
+    - Telco
+  summary: Block phone
+  description: Blocks a phone and suspend related SIM
```
I don't know if the sim is also suspended

bmoelans replied:
Had the same remark myself but forgot to press 'comment'.
It is not done as such by mv-web, but it seems so logic for me.
Block phone has immediate effect (and can be undone)

It is something I will take up in the analysis to discuss with care.

bmoelans replied:
Typo, I mean sim block, there are rare scenarios you want to block a phone but not suspend the sim.
Maybe this should be an extra boolean field although you just could do the extra call.

freeduerinckx replied:
While I do agree I would check this with Jens A or Stephanie just to be sure that the requirements are clear.

jaernouts replied:
That's actually a very good question. I think how it's working now, is that the Zendesk ticket is just copied to a wholesale ticket, and they handle it further down the line. I don't really know what wholesale does when running that request, I think it's best to check that with Jeroen Moors...

RoelandMatthijssens commented on paths/telco/block_phone.yaml:
```diff
@@ -0,0 +1,27 @@
+  tags:
+    - Telco
+  summary: Block phone
+  description: Blocks a phone and suspend related SIM
```
What do you have against articles? :p `suspends the related SIM`

bmoelans commented:
@RoelandMatthijssens pushed a fixup addressing your remarks

Further @freeduerinckx @jaernouts @Timvde customer care brought to my attention that a phone can have multiple IMEI number, e.g. some dual sim phone not all.

Also discussed some stuff with Jeroen Moors and concluded that it is not a good idea to automatically suspend the sim but it is a good idea to add it as boolean field. Certainly with the extra comment for the case of dual sim phones.

happy to hear your feedback

bmoelans commented:
@Kjoep and @Akheraphin any remarks from you side on this new end-point to block phone?
